3rdparty/voice_text/launch/voice_text.launch

```diff
@@ -7,6 +7,8 @@
   <arg name="use_machine" default="true" />
   <machine if="$(arg use_machine)" name="localhost" address="localhost" />
 
+  <arg name="sound_play_respawn" default="false" />
```

Comment:
is there any problem if set default=true ?
Cc: @furushchev

Reply:
Thanks for reviewing!
I have no problem.

How about you? @furushchev

Reply:
@k-okada @snozawa It's better for me to enable respawning by default. :+1:

Reply:
OK, I'll update this PR to do so!